src/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -27,3 +27,15 @@ Filecoin Storage Mining Subsystem
 # Sector in StorageMiner State Machine (both)
 
 {{< diagram src="diagrams/sector_fsm.dot.svg" title="Sector State Machine (both)" >}}
+
```

Comment:
We can remove this section altogether.

Reply:
moved it to the election-post.md. Thought it useful